lib/ex_force/client/tesla/tesla.ex

```diff
@@ -41,20 +41,24 @@ defmodule ExForce.Client.Tesla do
   end
 
   def build_client(instance_url, opts) when is_binary(instance_url) do
-    Tesla.client(
-      [
-        {ExForce.Client.Tesla.Middleware,
-         {instance_url, Keyword.get(opts, :api_version, @default_api_version)}},
-        {Tesla.Middleware.Compression, format: "gzip"},
-        {Tesla.Middleware.JSON, engine: Jason},
-        {Tesla.Middleware.Headers, get_headers(opts)}
-      ],
-      Keyword.get(opts, :adapter)
-    )
+    custom_middleware = get_from_opts_or_config(opts, :middleware, [])
```

Comment:
This middleware is always appended - so it would be better to make that explicit. What would be the better option name? `append_middleware`?

Reply:
Sure - I can change that name and also only scope it to `ExForce.Client.Tesla`.

Reply:
Updated!